src/azure-cli/azure/cli/command_modules/netappfiles/_exception_handler.py

```diff
@@ -3,13 +3,16 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+# pylint: disable=line-too-long
+
 from knack.util import CLIError
 from msrest.exceptions import ValidationError  # pylint: disable=import-error
-from msrestazure.azure_exceptions import CloudError
+from azure.core.exceptions import ClientAuthenticationError, HttpResponseError, ResourceExistsError, ResourceNotFoundError
```

Comment:
Use `HttpResponseError` is enough as it's the base class for other errors.

Reply:
Thanks, will fix it now